parsl/channels/local/local.py

```diff
@@ -64,12 +65,16 @@ def execute_wait(self, cmd, walltime=None, envs={}):
                 shell=True,
                 preexec_fn=os.setpgrp
             )
+            logger.debug("Created process with pid %s. Performing communicate", proc.pid)
             (stdout, stderr) = proc.communicate(timeout=walltime)
             retcode = proc.returncode
+            logger.debug("Process returned %s", proc.returncode)
 
-        except Exception as e:
-            logger.warning("Execution of command '{}' failed due to \n{}".format(cmd, e))
+        except Exception:
+            logger.exception(f"Execution of command '{cmd}' failed")
```

Comment:
For clarity in the logs, you might move the command to the end of the string, perhaps delineated by a newline.  Consider:
```python
logger.exception(f"Execution of command failed:\n  {cmd}")
```

Reply:
rearranged like that in 9745d88